doc/api-documentation.rst

```diff
@@ -37,6 +37,13 @@ API Documentation
     :undoc-members:
     :show-inheritance:
 
+:mod:`luma.core.interface.serial`
+"""""""""""""""""""""""
```

Comment:
Think the underline needs to be the same length as the title else doc generation will complain. we should probably add docgen to the toxic was task if possible